tools/src/main/java/org/apache/kafka/tools/VerifiableConsumer.java

```diff
@@ -663,16 +662,17 @@ public static void main(String[] args) {
         ArgumentParser parser = argParser();
         if (args.length == 0) {
             parser.printHelp();
-            Exit.exit(0);
+            System.exit(0);
         }
         try {
             final VerifiableConsumer consumer = createFromArgs(parser, args);
-            Exit.addShutdownHook("verifiable-consumer-shutdown-hook", () -> consumer.close());
+            Runtime.getRuntime().addShutdownHook(new Thread(consumer::close, "verifiable-consumer-shutdown-hook"));
 
             consumer.run();
         } catch (ArgumentParserException e) {
             parser.handleError(e);
-            Exit.exit(1);
+            //Can't use `Exit.exit` here because `Exit` doesn't exists on old version of Kafka.
```

Comment:
Can we add this comment in every usage? Also, let's be clear about the minimum version that this class must support in the comment. Same for VerifiableProducer.

Reply:
Fixed. 

With this change, `upgrade_test.py` passes for 0.8.8.2. I will attach results in a couple of hours.

Reply:
Seems reasonable to do this in Verifiable{Producer,Consumer}, yes; the changes related to this in these two classes LGTM.